packages/sdk/identity/src/offchain/accessors/binary.ts

```diff
@@ -45,6 +45,10 @@ export class PrivateBinaryAccessor implements PrivateAccessor<Buffer> {
     return writeEncrypted(this.wrapper, this.dataPath, data, toAddresses, symmetricKey)
   }
 
+  allowAccess(toAddresses: Address[], symmetricKey?: Buffer) {
```

Comment:
how come an `async` isn't needed here?

Reply:
Also, could you add a test in `offchain-data-wrapper.test.ts` that uses this function?

Reply:
hm seems like there should be a `async`, but it seemed to be working without. I've added it anyways